vehicle/views.py

```diff
@@ -0,0 +1,105 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from vehicle.models import Vehicle, Position,Journey
+from vehicle.serializers import VehicleSerializer, PositionSerializer , JourneySerializer
+
+
+@api_view(['GET', 'POST'])
```

Comment:
@tudor96, @daryne77, why not use `viewsets` ? http://www.django-rest-framework.org/api-guide/viewsets/